applications/ParticleMechanicsApplication/custom_conditions/grid_based_conditions/mpm_grid_surface_load_condition_3d.cpp

```diff
@@ -188,8 +188,8 @@ void MPMGridSurfaceLoadCondition3D::CalculateAll(
 {
     KRATOS_TRY;
```

Comment:
What changes would be required to use this condition instead? https://github.com/KratosMultiphysics/Kratos/blob/feature/iga_brep_topology_kernel/applications/IgaApplication/custom_conditions/load_condition.cpp